composer.json

```diff
@@ -32,7 +32,8 @@
         "league/flysystem": "~1.0",
         "php-di/php-di": "^5.0",
         "league/event": "^2.1",
-        "symfony/validator": "^2.7"
+        "symfony/validator": "^2.7",
+        "phpdocumentor/flyfinder": "dev-master@dev"
```

Comment:
I have just merged an update to flyfinder, when you merge develop into this PR: can you also update this dependency?